rust-keybroker/keybroker-server/src/error.rs

```diff
@@ -57,6 +57,14 @@ pub enum VerificationErrorKind {
     /// It was not possible to find an appraisal policy for the evidence type
     #[error("No appraisal policy was found for the evidence type.")]
     PolicyNotFound,
+
+    /// There are no known-good RIM values
+    #[error("No known-good RIM values.")]
+    NoRIMValues,
```

Comment:
Getting a bit picky now, sorry, but can we align to `ReferenceValues` everywhere instead of RIM (which stands for Realm Initial Measurement, so is again CCA specific)

Reply:
This is fixed in the latest iteration of the patch set !